### ISSUE TITLE
vaex-distributed examples Unable to find？

### ISSUE THREAD

**Issue by fuhao009** [vaexio/vaex#655]
Can you provide a use case for vaex-distributed？

**Comment by JovanVeljanoski** [comment]
I don't believe we have an published examples on this. It was more done as a proof of concept, in order for us to determine how to develop vaex in the future. 

Perhaps @maartenbreddels can give you a very simple usecase.

Just curious: do you have a particular need for distributed computing, or do you just want to try things out?

**Comment by kyprifog** [comment]
I'm also curious what the philosophy for vaex-distributed is.  All the examples I see use big machines, is that where vaex is focused mostly?

**Comment by maartenbreddels** [comment]
Hi,

we thought a lot about vaex-distributed. Our current position is that you probably don't need it (get a bigger instance/faster storage) and you are good to go. The extra support burden at this moment for open source usage is too big, so we develop this in vaex-enterprise (closed source). I can imagine we can have research and university friendly licenses in the future. Hope you understand our position, feedback welcome though!

Regards,

Maarten
